README.md

```diff
@@ -49,6 +49,8 @@ Set the **Homepage URL** to the external URL of your app e.g.
 
 use an address like `https://abcd-123-45-67-89.ngrok.io`.
 
+Skip the "Callback URL" section.
```

Comment:
I was expecting to need to specify a Callback URL for an oAuth application, so I felt like this could address that question.